app/views/admin/locations/index.html.erb

```diff
@@ -1,14 +1,30 @@
 <p style="color: green"><%= notice %></p>
 
-<h1>Locations</h1>
-
 <div id="locations">
-  <% @locations.each do |location| %>
-    <%= render location %>
-    <p>
-      <%= link_to "Show this location", admin_location_path(location) %>
-    </p>
-  <% end %>
-</div>
-
-<%= link_to "New location", new_admin_location_path %>
+  <div class="container">
+    <h1>Locations</h1>
+    <table class="table table-striped table-bordered table-sm">
+      <thead class="table-dark">
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Latitude</th>
+          <th scope="col">Longitude</th>
+          <th scope="col">Watch location</th>
+        </tr>
+      </thead>
+      <% @locations.each do |location| %>
+        <tbody>
+          <tr>
+            <th scope="row"><%= @locations.ids.index{|x| x == location.id} + 1 %></th>
```

Comment:
@lime666 you can use [each_with_index](https://apidock.com/ruby/Enumerator/each_with_index) method to avoid complicated logic. for example, `@locations.each.with_index(1) do |location, index|` and use `index` instead of `@locations.ids.index{|x| x == location.id} + 1`.